src/main/java/org/jabref/model/groups/AutomaticGroup.java

```diff
@@ -2,7 +2,10 @@
 
 import java.util.Set;
 
+import javafx.collections.ObservableList;
```

Comment:
I know this is convenient, but I am strongly against it. We do not want the model to be dependent on a GUI technology. If you want this conversion to be done based on `ObservableList`, it will have to be done elsewhere.